Docker/Jenkins2/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM jenkins/jenkins:2.233
+
+USER root
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# install python and pip and aws cli
+RUN set -xe && apt-get update && apt-get install -y apt-utils dnsutils python python-setuptools python-dev python-pip python3 python3-pip build-essential zip unzip jq less vim gettext-base
+RUN set -xe && python -m pip install awscli --upgrade && python -m pip install pytest --upgrade && python -m pip install PyYAML --upgrade
+RUN set -xe && python3 -m pip install pytest --upgrade && python3 -m pip install PyYAML --upgrade
+RUN set -xe && python -m pip install yq --upgrade && python3 -m pip install yq --upgrade
```

Comment:
Can this be simplified? Also, why you need multiple Python's versions and packages?

Note, shell can be simplified like this. Put after `FROM`.
```
SHELL ["/bin/bash", "-xe", "pipefail", "-c"]
```

Reply:
This is a copy `ipsi literis` from the existing instruction in the original Jenkins1 Dockerfile.
It should be a `TODO`. But, for now, I guess it's ok to have the first set of artifacts for Jenkins2 officially merged into `cloud-automation`.